elements/pfe-tabs/src/pfe-tab.scss

```diff
@@ -59,10 +59,12 @@ $pfe-tabs__tab--PaddingBottom-factor:  1.5; // 54px at 16px base.
   background-color: #{pfe-color(surface--border--darkest)};
 }
```

Comment:
I'm trying to decide if its a good idea to worry about it in this MR (probably not, since this is the combo) but I'm brain-dumping just for the moment:

We'll want to update the tab component to include some local variables, and we'll probably want to support the other theme palette colors. So something like this:

```scss
:host {
  --pfe-tab--color--bg:            #{pfe-color(surface--darkest)};
  --pfe-tab--color--text:          #{pfe-color(surface--darkest--text)};
}
```

further down:
```scss
:host([pfe-variant="secondary"][color="accent"]) {
   --pfe-tab--color--bg:  #{pfe-color(surface--accent)};
   --pfe-tab--color--text: #{pfe-color(surface--accent--text)}
}
:host([pfe-variant="secondary"][color="complement"]) {
   --pfe-tab--color--bg:    #{pfe-color(surface--complement)};
   --pfe-tab--color--text:  #{pfe-color(surface--complement--text)}
}
/* etc */
```

Reply:
https://github.com/patternfly/patternfly-elements/issues/272